src/models/Tosser.js

```diff
@@ -76,7 +77,16 @@ class Tosser {
         } else if (typeof strategy === 'string') {
           const handlerFun = handler.getRetryHandler(strategy);
           noRetry = handlerFun(ctx);
-        } else {
+        } else if (status) {
+          if (Array.isArray(status)) {
+            noRetry = status.toString().split(",").includes(ctx.res.statusCode.toString());
+            console.log("if:"+noRetry)
+          } else {
+            noRetry = (status == ctx.res.statusCode);
```

Comment:
same logic here. use `===`

```suggestion
            noRetry = !(status === this.response.statusCode);
```